src/Providers/AddonServiceProvider.php

```diff
@@ -47,26 +50,27 @@ public function boot()
             }
 
             $this
-                ->bootEvents()
-                ->bootTags()
-                ->bootScopes()
                 ->bootActions()
+                ->bootCommands()
+                ->bootConfig()
+                ->bootEvents()
+                ->bootFieldsets()
                 ->bootFieldtypes()
+                ->bootMiddleware()
                 ->bootModifiers()
-                ->bootWidgets()
-                ->bootCommands()
-                ->bootSchedule()
                 ->bootPolicies()
-                ->bootStylesheets()
-                ->bootScripts()
                 ->bootPublishables()
-                ->bootConfig()
-                ->bootTranslations()
+                ->bootPublishAfterInstall()
                 ->bootRoutes()
-                ->bootMiddleware()
+                ->bootSchedule()
+                ->bootScopes()
+                ->bootScripts()
+                ->bootStylesheets()
+                ->bootTags()
+                ->bootTranslations()
                 ->bootUpdateScripts()
                 ->bootViews()
-                ->bootPublishAfterInstall();
+                ->bootWidgets();
```

Comment:
Please revert the re-ordering of these lines. Just include any new or removed methods.

Unless of course there's a reason for reordering.

Reply:
> Unless of course there's a reason for reordering.

My sanity.